src/main/java/command/QuizCommand.java

```diff
@@ -0,0 +1,56 @@
+package command;
+import Dictionary.WordBank;
+import exception.WordBankNotEnoughForQuizException;
+
+import java.util.*;
+
+/**
+ * Generate a 4-option MCQ. User should answer by index of options, namely 1, 2, 3, 4.
+ */
+public class QuizCommand extends Command  {
+    public String question;
+    public String answer;
+    public String[] options;
+    protected Random random;
+    protected ArrayList<Integer> randomNumbers;
+    public int optionSequence;
+
+    public QuizCommand(){
+        this.random = new Random();
+        this.options = new String[4];
```

Comment:
Good to put this 4 as a private static final attribute. That way you can easily alter it if you want to change to 5-option MCQ etc